packages/rpc-engine/middlewares/validate-rpc-data.js

```diff
@@ -99,7 +99,9 @@ function validateNetworkSupport({req}) {
   const {method, network} = req
 
   const bothSupport =
-    method.startsWith('wallet') || method.startsWith('personal')
+    method.startsWith('wallet') ||
+    method.startsWith('personal') ||
+    method === 'eth_signTypedData_v4'
```

Comment:
We can add a description in zendesk developer document to let developer know that we also support `cfx_signTypedData_v4` and `eth_signTypedData_v4`, which follow different proposal.